x-pack/legacy/plugins/ingest/index.ts

```diff
@@ -22,6 +23,12 @@ export function ingest(kibana: any) {
     config: () => config,
     configPrefix: CONFIG_PREFIX,
     uiExports: {
+      savedObjectSchemas: {
```

Comment:
For my information what this is doing?

Reply:
This marks this saved object as working across all kibana spaces. It also saves this data to a custom index rather then .kibana 

All the saved object data types we use and create will need these settings